.microcks/overlays.yaml

```diff
@@ -0,0 +1,204 @@
+overlay: 1.0.0
+info:
+  title: Train Travel API - Microcks
+  description: |
+    This API is a mock server for the Train Travel API, which provides a way to
+    find and book train trips across Europe. It includes endpoints for finding
+    stations, getting timetables, and managing bookings.
+  version: 1.0.0
+
+actions:
+
+  - target: $["x-topics"]
+    description: Remove x-topics, not needed for mock server.
+    remove: true
+
+  - target: $.paths["/stations"].get.responses["200"].content["application/json"]["example"]
+    description: Remove the example from get /stations.
+    remove: true
+
+  - target: $.paths["/stations"].get.responses["200"].content["application/json"]
+    description: Add Microcks examples for get /stations.
+    update:
+      examples:
+        stations:
+          summary: A list of train stations
+          value:
+            data:
+              - id: efdbb9d1-02c2-4bc3-afb7-6788d8782b1e
+                name: Berlin Hauptbahnhof
+                address: Invalidenstraße 10557 Berlin, Germany
+                country_code: DE
+                timezone: Europe/Berlin
+              - id: b2e783e1-c824-4d63-b37a-d8d698862f1d
+                name: Paris Gare du Nord
+                address: 18 Rue de Dunkerque 75010 Paris, France
+                country_code: FR
+                timezone: Europe/Paris
+            links:
+              self: https://api.example.com/stations&page=2
+              next: https://api.example.com/stations?page=3
+              prev: https://api.example.com/stations?page=1
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="origin")].example
+    description: Remove the example from get /trips - origin parameter.
+    remove: true
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="origin")]
+    description: Add Microcks examples for get /trips - origin parameter
+    update:
+      examples:
+        trips:
+          value: 'b2e783e1-c824-4d63-b37a-d8d698862f1d'
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="destination")].example
+    remove: true
+
+  - target: $.paths["/trips"].get.parameters[?(@.name=="destination")]
+    update:
+      examples:
+        trips:
+          value: '2024-02-01T09:00:00Z'
+
```

Comment:
I believe this should change the `date` parameter (looking at the value). Or you expected to change the `destination` parameter and the value should be Berlin Bahnhof? (`efdbb9d1-02c2-4bc3-afb7-6788d8782b1e`)?